modules/client/src/tokenVoting/internal/client/encoding.ts

```diff
@@ -9,46 +9,46 @@ import {
 } from "../../../client-common";
 import { isAddress } from "@ethersproject/address";
 import {
-  IClientErc20Encoding,
-  IErc20PluginInstall,
+  ITokenVotingClientEncoding,
+  ITokenVotingPluginInstall,
   IMintTokenParams,
 } from "../../interfaces";
-import { ERC20_PLUGIN_ID } from "../constants";
+import { TOKEN_VOTING_PLUGIN_ID } from "../constants";
 import {
-  ERC20Voting__factory,
-  IERC20MintableUpgradeable__factory,
+  TokenVoting__factory,
+  ITokenMintableUpgradeable__factory,
 } from "@aragon/core-contracts-ethers";
-import { erc20InitParamsToContract, mintTokenParamsToContract } from "../utils";
+import { tokenVotingInitParamsToContract, mintTokenParamsToContract } from "../utils";
 /**
- * Encoding module the SDK ERC20 Client
+ * Encoding module the SDK TokenVoting Client
  */
-export class ClientErc20Encoding extends ClientCore
-  implements IClientErc20Encoding {
+export class TokenVotingClientEncoding extends ClientCore
+  implements ITokenVotingClientEncoding {
   constructor(context: ContextPlugin) {
     super(context);
-    Object.freeze(ClientErc20Encoding.prototype);
+    Object.freeze(TokenVotingClientEncoding.prototype);
     Object.freeze(this);
   }
```

Comment:
(unrelated)
Wasn't this globally undone in the end?

Reply:
Nope. It was decided to keep it